x-pack/plugins/lists/server/services/items/delete_list_item.ts

```diff
@@ -36,6 +38,19 @@ export const deleteListItem = async ({
       },
       refresh,
     });
+
+    if (response.deleted) {
+      const checkIfListItemDeleted = async (): Promise<void> => {
+        const deletedListItem = await getListItem({ esClient, id, listItemIndex });
+        if (deletedListItem !== null) {
+          throw Error('List item has not been re-indexed in time');
```

Comment:
This message is a little odd out of context: _why_ does it matter that a list item hasn't been re-indexed? Also, is `reindex` the right term, or should it be `refresh`?

```suggestion
          throw Error('List item was deleted, but the change was not propagated in the expected time interval.');
```